dev/monitoring/datarepo-monitoring.yaml

```diff
@@ -0,0 +1,108 @@
+---
+create-secret-manager-secret:
+  enabled: true
+  secrets:
+    - secretName: grafana-google-client-secret
+      vals:
+        - kubeSecretKey: GF_AUTH_GOOGLE_CLIENT_SECRET
+          path: secret/dsde/datarepo/dev/clientid-secret.json
+          vaultKey: oauthClientSecret
+
+backend-config:
+  enabled: true
+
+grafana-gcp-managed-cert:
+  enabled: true
+  fullnameOverride: grafana-gcp-managed-cert
+  domainName: grafana.datarepo-dev.broadinstitute.org
+
+prometheus-gcp-managed-cert:
+  enabled: false
+  fullnameOverride: prometheus-gcp-managed-cert
+  domainName: data-prometheus.datarepo-dev.broadinstitute.org
+
+kube-prometheus-stack:
+  enabled: true
+  prometheusOperator:
+    nodeSelector:
+      cloud.google.com/gke-nodepool: dev-node
+    admissionWebhooks:
+      patch:
+        nodeSelector:
+          cloud.google.com/gke-nodepool: dev-node
+    createCustomResource: false
+  prometheus:
+    serviceAccount:
+      annotations:
+        iam.gke.io/gcp-service-account: sidecar-test-mark@broad-jade-dev.iam.gserviceaccount.com      # workloadID annotation for sidecar to write to stackdriver
```

Comment:
Do we want a more explicit service account here?  Probably yes

Reply:
This one is a test account till we rebuild dev, it's fine as is.